### PR TITLE
Add SQL Quiz Question "How to Access All Column Names"

### PR DESCRIPTION
**Pull request by johnpatrickanders**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -355,6 +355,17 @@ const sqlQuiz = [
...
   },
+  {
+    Question:
+      "In SQL, where can we access all column names within a database?",
+    Answer: "INFORMATION_SCHEMA.COLUMNS",
+    Distractor1: "COLUMNS",
+    Distractor2: "SCHEMA.COLUMNS",
+    Distractor3: "SCHEMA.INFO.COLS",
+    Explanation:
+      "You can select from INORMATION_SCHEMA.COLUMNS and combine with a WHERE clause when needing to identify where a certain column is stored.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:366]: spelling error

```suggestion
      "You can select from INFORMATION_SCHEMA.COLUMNS and combine with a WHERE clause when needing to identify where a certain column is stored.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Hi @johnpatrickanders ! 

Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by johnpatrickanders** [comment]
> Hi @johnpatrickanders !
> 
> Congrats on your first contribution to our repository. 🎉 We look forward to future contributions. 👍

Thanks so much, @jdwilkin4!

**Pull request merged**